plonetheme/barceloneta/theme/less/barceloneta-compiled.css

```diff
@@ -2397,6 +2394,9 @@ fieldset[disabled] .link-parent.active {
   padding: 0;
   font-weight: 300;
   /* wanna hide the label "next ... items"? */
+  /* .label {
```

Comment:
Was this comment pushed by accident?

Reply:
oh I see this is the result of the compilation, right?So it has nothing to do with the PR that indeed fixes the variable definition.